### ISSUE TITLE
Examples?

### ISSUE THREAD

**Issue by Rush** [cartant/rxjs-etc#76]
Awesome library. Could you provide some examples for each operators?

**Comment by cartant** [comment]
Nope. I simply don't have time for that. Behaviour can be inferred from the tests.
